genie-agent/src/main/java/com/netflix/genie/agent/rpc/GrpcConfig.java

```diff
@@ -46,7 +46,7 @@ ManagedChannel channel(
         final ArgumentDelegates.ServerArguments serverArguments
     ) {
         return ManagedChannelBuilder.forAddress(serverArguments.getServerHost(), serverArguments.getServerPort())
-            .usePlaintext(true)
+            .usePlaintext()
```

Comment:
This is a change of API in the new gRPC library.
Turning TLS on and off via configuration is a must at some point (i.e. when we add metatron, since it will require other parameters like a path to a keystore/cert/whatever)